src/signaturelist/signaturelist.js

```diff
@@ -0,0 +1,28 @@
+var signatureDataBase = {'Steve Jobs':'"Stay hungry, stay foolish." -Steve Jobs',
```

Comment:
Signature database currently in JS function. This means this is a in-memory structure which gets recreated everytime we load the addin. We need a persistent way to store this database outside the JS file - example: localstorage, caching, sqlite

Reply:
Hi @jruiz50 - are you done moving the signatureDB to localStorage? I can't see that code in the PR yet. If you have it working, please commit/push and we can review that piece of code too.